### PR TITLE
Fixing #461 - warning with a reference to non existing location in source file

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

**Comment by forki** [comment]
ready for review

File: tests/fsharpqa/Source/EntryPoint/W_NoEntryPointTypeInNamespace.fs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Empty program entry point warning
 // Lack of entry point produces warning with correct source location when compiled to *.exe
 // when source file declares type inside namespace
-//<Expects id="FS0988" span="(1,81-1,81)" status="warning">Main module of program is empty: nothing will happen when it is run</Expects>
+//<Expects id="FS0988" span="(10,18-10,18)" status="warning">Main module of program is empty: nothing will happen when it is run</Expects>
```
Comment (dsyme) [inline tests/fsharpqa/Source/EntryPoint/W_NoEntryPointTypeInNamespace.fs]: Agreed we shouldn't report a non-existent location, but we need to report this on some actual source text, not a single character

Comment (forki) [inline tests/fsharpqa/Source/EntryPoint/W_NoEntryPointTypeInNamespace.fs]: it's at https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/IlxGen.fs#L5912

**Comment by forki** [comment]
ready for review

**Comment by KevinRansom** [comment]
@dotnet-bot test this please

**Comment by KevinRansom** [comment]
@forki 

Thanks for taking care of this

Kevin

**Pull request merged**